app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
```

Comment:
Consider providing more descriptive and meaningful names.
In your case 'vstr' is not descriptive
Maybe it is better to use:
```suggestion
    def visit_cafe(self, visitors: dict) -> str:

```